webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -3569,6 +3569,14 @@ const firstVisibleMsg = {
                 ).style.display = 'none';
             };
 
+            window.onkeyup= function(event){
+              if(event.keyCode==27) {
+                parent.document.getElementById(
+                    'km-fullscreen-image-modal'
+                ).style.display = 'none';
+              }
+            }
+
```

Comment:
@thakaresakshi121 If any user is using window.onkeyup in their code, it will either overwrite their existing listener or remove our listener.

Try to use the event in the element instead of the window object.